device-types/iXsystems/truenas-mini-r.yaml

```diff
@@ -0,0 +1,38 @@
+---
+manufacturer: iXsystems
+model: TrueNAS Mini R
+slug: ixsystems-truenas-mini-r
+u_height: 2
+is_full_depth: true
+front_image: true
+rear_image: true
+airflow: front-to-rear
+comments: '[Basic Setup Guide](https://www.truenas.com/docs/files/MiniRBSG1.0.pdf)'
+weight: 18.7
+weight_unit: lb
+console-ports:
+  - name: VGA
+    type: other
+  - name: USB 2.0 - 1
+    type: usb-a
+  - name: USB 2.0 - 2
+    type: usb-a
+  - name: USB 3.0 - 1
+    type: usb-a
```

Comment:
Are these USB ports actually used for console access? If not, please remove their defintions